codespan/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.2] - 2020-10-XX
+
```

Comment:
Might be good to note the version jump here, like with `codespan-reporting`

Reply:
done for both